curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-recursion-by-building-a-decimal-to-binary-converter/63db7f4677d06d7500a13321.md

```diff
@@ -9,6 +9,8 @@ dashedName: step-1
 
 In this project, you'll build a decimal and binary converter and learn about both number systems. You'll also learn about recursion by using it to perform the conversions.
 
+`Note:` this project will only convert positive numbers into binary.
```

Comment:
I don't think this note is necessary. But more importantly, never use backticks like this. backticks are only for code and words that are not to be translated.